practice-vue-router-answer-2/src/App.vue

```diff
@@ -1,29 +1,24 @@
-<script>
-import {ref, onMounted} from "vue"
-import {RouterView} from "vue-router"
-export default {
-  /**
-   * コンポーネントをローカル登録することによって、登録したコンポーネントをテンプレートで使用できるようになります。
-   * 参考: https://v3.ja.vuejs.org/guide/component-registration.html#%E3%83%A2%E3%82%B7%E3%82%99%E3%83%A5%E3%83%BC%E3%83%AB%E3%82%B7%E3%82%B9%E3%83%86%E3%83%A0%E5%86%85%E3%81%AE%E3%83%AD%E3%83%BC%E3%82%AB%E3%83%AB%E7%99%BB%E9%8C%B2
-   */
-  components: {
-    RouterView
-  },
-  setup() {
-    const location = ref(null)
-    onMounted(() => {
-      location.value = window.location
-    })
-    return {location}
-  }
-}
+<script setup>
+import { ref, onMounted } from "vue";
+/**
+ * コンポーネントをローカル登録することによって、登録したコンポーネントをテンプレートで使用できるようになります。
+ * 参考: https://v3.ja.vuejs.org/guide/component-registration.html#%E3%83%A2%E3%82%B7%E3%82%99%E3%83%A5%E3%83%BC%E3%83%AB%E3%82%B7%E3%82%B9%E3%83%86%E3%83%A0%E5%86%85%E3%81%AE%E3%83%AD%E3%83%BC%E3%82%AB%E3%83%AB%E7%99%BB%E9%8C%B2
+ */
```

Comment:
setup 関数の説明が当てはまらないので対応が必要

```suggestion
/**
 * `<script setup>`  を使用する場合、 `<script setup>` 内で宣言された
 * トップレベルのバインディング（変数、関数宣言、インポートを含む）は、テンプレートで直接使用できます
 * 引用:  https://ja.vuejs.org/api/sfc-script-setup.html#basic-syntax
 */
```